public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx

```diff
@@ -65,6 +65,10 @@ export const SingleTopBar = memo(function SingleTopBar({
   const isLargeScreen = useMediaQueryMinWidth('lg');
   const topLevelScopes = !showToolbarLevel && isLargeScreen && scopes?.state.enabled;
 
+  const onHomeClicked = () => {
+    reportInteraction('grafana_home_clicked', { url: homeNav?.url });
```

Comment:
Why do we need to capture the homeNav url? I assume it'll always be the same?

Reply:
good point